kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -467,7 +489,7 @@ class SolvingStrategy
     ///@name Member Variables
     ///@{
 
-    ModelPart& mrModelPart;
+    ModelPart* mpModelPart;
```

Comment:
this is a delicate change

I think this is one of the reasons @pooyan-dadvand suggested to use variadic templates to pass different number of arguments to the `Create`

Reply:
It is ok as long as we are only using the `GetModelPart` method.